cogniac/edgeflow.py

```diff
@@ -350,3 +350,49 @@ def get_next(url):
                 if limit and count == limit:
                     return
             url = resp['paging'].get('next')
+
+
+    def get_aggregated_stats(self, start=None, end=None):
+        """
+        Returns total detections and pixels processed 
+        between start and end timestamp (default: last 5 minutes)
+
+        start (float)    filter by last update 
+                         timestamp > start (seconds since epoch)
+        end (float)      filter by last update
+                         timestamp < end   (seconds since epoch)
+        """
+        if end is None:
+            end = time()
+        if start is None:
+            start = end - 300
+        events = self.status(subsystem_name='model_detections*',
+                             start=start,
```

Comment:
this filters based on cloudtime timestamp, which can be delayed relative to actual edgeflow timestamp by some arbitrary time if the connection to cloudcore is slow or temporarily disconnected

Reply:
In that case, we might have to fetch all the event reports that has happened after the start timestamp on that EF. This will make the code super slow.